orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -272,6 +272,27 @@ def test_masked_select_broadcast(self):
         assert torch.allclose(cpu_result, ort_result.cpu())
         assert cpu_result.dim() == ort_result.dim()
 
+    def test_bitwise_and(self):
+        device = self.get_device()
+        cpu_a = torch.tensor([[0], [1], [1]], dtype=bool)
+        cpu_b = torch.tensor([[1], [0], [1]], dtype=bool)
+        ort_a = cpu_a.to(device)
+        ort_b = cpu_b.to(device)
+        cpu_result = torch.bitwise_and(cpu_a, cpu_b)
+        ort_result = torch.bitwise_and(ort_a, ort_b)
```

Comment:
add the out= param and verify those as well please.

Reply:
Verified